Resources/Locale/en-US/_NF/flavors/flavor-profiles.ftl

```diff
@@ -0,0 +1,2 @@
+flavor-base-basic = basic
+flavor-base-fast = fast
```

Comment:
Nitpick - I'd think "fast" would be a complex flavour (be sure to change it elsewhere if taking this).  I would've made basic a complex flavour, but "flavor-complex-basic" gives me an aneurysm.
```suggestion
flavor-complex-fast = fast
```